packages/align-deps/src/presets/microsoft/react-native/profile-0.73.ts

```diff
@@ -33,6 +33,11 @@ export const profile: Profile = {
     version: "^0.73.0",
     capabilities: ["react"],
   },
+  "core-visionos": {
+    name: "#meta",
```

Comment:
Did we agree to use Callstack's as default?
```suggestion
    name: "@callstack/react-native-visionos",
```